cves/CVE-2013-4316.yml

```diff
@@ -60,13 +66,13 @@ fixes_vcc_instructions: |
   Please put the SVN commit number in "commit" below, and any notes about how this
   was discovered in the "note" field.
 fixes:
-  - commit: 58947c3f85ae641c1a476316a2888e53605948d1
+  - commit: c643336945dda84cbcdc8a39530baa24fede28c4
     note: Disables allowing dynamic method calls by default
   - commit:
     note:
 vccs:
-  - commit: 20da1d6161f309dea758e3b3f9458ed7afd96f18
-    note: Allowed dynamic method calls by default
+  - commit: c95a0f8b5e93bfb866a5095b00dd33950d2187f1
+    note: Added support for dynamic method calls, was enabled by default
```

Comment:
Is this the exact commit message? If so, maybe put it in quotes and reword the purpose of the commit. I'm unclear on what the note means. Was it the support for the DMI that was enabled by default, or was it the actual DMI?

Reply:
I've clarified this bit as well as added in the actual commit message